webapp/src/Doctrine/DBAL/Types/JsonType.php

```diff
@@ -7,16 +7,12 @@
 use Doctrine\DBAL\Types\JsonType as BaseJsonType;
 
 /**
- * Class JsonType
- *
  * We overwrite this class because we want to preserve zero fractions when
  * storing JSON data in the database. Otherwise "5.0" will be converted to "5",
  * which gives some inconsistencies between the API and event feed.
  *
  * Also we always want a LONGTEXT field and not a JSON field, as that is only
  * supported by MySQL 5.7+.
```

Comment:
Not part of your PR but I think we shouldn't target MySQL <5.7 anymore.

Reply:
Mariadb doesn’t have json either

Reply:
Ok, thats a good reason to not change the code.